apollo-openapi/src/main/java/com/ctrip/framework/apollo/openapi/api/NamespaceOpenApiService.java

```diff
@@ -26,9 +26,9 @@
  */
 public interface NamespaceOpenApiService {
 
-  OpenNamespaceDTO getNamespace(String appId, String env, String clusterName, String namespaceName);
+  OpenNamespaceDTO getNamespace(String appId, String env, String clusterName, String namespaceName, boolean fillItemDetail);
```

Comment:
As it's a public API, we cannot alter the existing signature, but we can add a new API.